.travis.yml

```diff
@@ -46,4 +46,13 @@ matrix:
     install:
       - pip install -r requirements.txt
       - pip install -r dev_tools/conf/pip-list-dev-tools.txt
-    script: check/pytest --ignore=cirq/contrib
+    script: check/pytest --ignore=cirq/contrib --benchmark-skip
+
+
+    - os: linux
+    env: NAME=Performance tests
+    python: "3.7"
+    install:
+      - pip install -r requirements.txt
+      - pip install -r dev_tools/conf/pip-list-dev-tools.txt
+    script: check/pytest --benchmark-only
```

Comment:
We should probably keep `--ignore=cirq/contrib` here.

Reply:
Left it like this if we want to benchmark conrib programs (Do we want to?)

Reply:
Our CI setup skips testing contrib for correctness (skipped above), so we should probably also skip testing contrib for performance.

Reply:
I disagree that we should skip contrib perf tests. The without contrib check is to ensure that we don't accidentally depend on contrib dependencies in the main codebase. We only need one check of this style.